Source/WebKit/Platform/cocoa/LayerHostingContext.mm

```diff
@@ -60,13 +68,29 @@
 #if PLATFORM(IOS_FAMILY) && !PLATFORM(MACCATALYST)
     // Use a very large display ID to ensure that the context is never put on-screen
     // without being explicitly parented. See <rdar://problem/16089267> for details.
-    layerHostingContext->m_context = [CAContext remoteContextWithOptions:@{
+    auto contextOptions = @{
         kCAContextSecure: @(options.canShowWhileLocked),
 #if HAVE(CORE_ANIMATION_RENDER_SERVER)
         kCAContextIgnoresHitTest : @YES,
         kCAContextDisplayId : @10000
 #endif
-    }];
+    };
+#if USE(EXTENSIONKIT)
+    if ([get_SEHostableClass() respondsToSelector:@selector(createHostableWithOptions:error:)]) {
```

Comment:
I think you filed a radar to move away from the dynamic `respondsToSelector` once the API is complete, right?